code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1099,15 +1099,9 @@
 /mob/living/carbon/human/species/abductor
 	race = /datum/species/abductor
 
-/mob/living/carbon/human/species/android
-	race = /datum/species/android
```

Comment:
> Unusued species

This one happens to be used, unless you apply some ""buffs"" to the robot species when converted in altair don't remove it.